docker/README.md

```diff
@@ -43,6 +43,34 @@ Once this is done you should run the following command to build `aarch64` static
 ./docker-static-build.sh aarch64
 ```
 
+### Packages from statically linked binaries
+
+It's possible to create packages (ubuntu/debian & fedora) using already
+built binaries. This means that the build step of the package will be reduced
+to copying the binaries instead of building again from scratch.
+
+Using Ubuntu/Debian and assuming all Tezos binaries are located in the `out` directory:
```

Comment:
This comment looks outdated now, same for Fedora packages
```

Reply:
Thanks, I updated the docs.